build.gradle

```diff
@@ -0,0 +1,60 @@
+buildscript {
+    ext.kotlin_version='1.2.30'
+    ext.gradle_docker_version='3.2.4'
+
+    repositories {
+        jcenter()
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath "com.bmuschko:gradle-docker-plugin:$gradle_docker_version"
+    }
+}
+
+plugins {
+    id 'org.jetbrains.kotlin.jvm' version '1.2.30'
+}
+
+description = 'SAML Conformance Test Kit'
+
+project.ext {
+    project_version='1.0-SNAPSHOT'
+    kotlin_test_version='2.0.7'
+    rest_assured_version='2.9.0'
+    org_slf4j_version='1.7.1'
+    junit_version='4.12'
+    guava_version='24.0-jre'
+}
+
+allprojects {
+    group = 'org.codice.samlconf'
+    version = "$project_version"
+}
+
+subprojects {
+    apply plugin: 'java'
+    apply plugin: 'maven'
+    apply plugin: 'kotlin-kapt'
+    apply plugin: 'kotlin'
+    apply plugin: 'com.bmuschko.docker-remote-api'
+
+    sourceCompatibility = 1.8
+    targetCompatibility = 1.8
+
+    repositories {
+        mavenLocal()
+        mavenCentral()
+    }
+
+    dependencies {
+        compile "org.jetbrains.kotlin:kotlin-reflect"
```

Comment:
❓ What is this dependency for? Looks like a new one.

Reply:
Yes, that was because of this warning we were getting:

```
w: Runtime JAR files in the classpath should have the same version. These files were found in the classpath:
    /Users/destab/.m2/repository/org/jetbrains/kotlin/kotlin-stdlib-jdk8/1.2.30/kotlin-stdlib-jdk8-1.2.30.jar (version 1.2)
    /Users/destab/.m2/repository/org/jetbrains/kotlin/kotlin-stdlib-jdk7/1.2.30/kotlin-stdlib-jdk7-1.2.30.jar (version 1.2)
    /Users/destab/.m2/repository/org/jetbrains/kotlin/kotlin-reflect/1.1.51/kotlin-reflect-1.1.51.jar (version 1.1)
    /Users/destab/.m2/repository/org/jetbrains/kotlin/kotlin-stdlib/1.2.30/kotlin-stdlib-1.2.30.jar (version 1.2)
w: Consider providing an explicit dependency on kotlin-reflect 1.2 to prevent strange errors
```

Reply:
Awesome! I was getting tired of seeing those.